entries/r/resend.com.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Resend": {
+    "domain": "resend.com",
+    "tfa": [
+      "totp"
+    ],
+    "categories": [
+      "communication"
```

Comment:
Is it not more of a developer product? Or perhaps both categories should be used?

Reply:
It is a developer product, but I put in communication because tools like SendGrid are in there. I'll add the developer category.